addons/product_catalog/__openerp__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    'name': 'Products Catalog',
+    'category': 'Sales Management',
+    'depends': ['product'],
+    'description': """
+Manage all aspects of Products from Product Catalog Menu
+========================================================
+
+Product Catalog supports variants, different unit of measures and properties.
```

Comment:
Change string into : It is the central place to manage your products, its variants, unit of measures and other properties.